### PR TITLE
[Components with more than 1 service] LPS-186836 Refactor components from multiple modules

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://issues.liferay.com/browse/LPS-186836

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:data-engine

**Comment by MrJohn1911** [comment]
ci:test:echo

**Comment by MrJohn1911** [comment]
ci:test:workflow

**Comment by MrJohn1911** [comment]
ci:test:lima

**Comment by MrJohn1911** [comment]
ci:test:analytics-cloud

**Review by rafaprax — APPROVED** [review]

**Comment by catarineaguiar** [comment]
ci:test:analytics-cloud-acceptance

**Comment by catarineaguiar** [comment]
no regressions found.
@MrJohn1911  if the analytis cloud acceptance suite pass, please add the qa passed label, thanks

**Review by dantewang — CHANGES_REQUESTED** [review]
@MrJohn1911 Please change as I commented, thanks!

File: modules/apps/product-navigation/product-navigation-personal-menu-web/src/main/java/com/liferay/product/navigation/personal/menu/web/internal/configuration/PersonalMenuConfigurationRegistryImpl.java [hunk 1]
```diff
@@ -36,72 +30,33 @@
...
-	}
+	implements PersonalMenuConfigurationRegistry {
 
 	@Override
 	public PersonalMenuConfiguration getCompanyPersonalMenuConfiguration(
 		long companyId) {
 
-		if (_companyConfigurationBeans.containsKey(companyId)) {
-			return _companyConfigurationBeans.get(companyId);
+		if (PersonalMenuConfigurationRegistryUtil.
```
[the remark this answers is not in the document]
Comment (dantewang) [reply modules/apps/product-navigation/product-navigation-personal-menu-web/src/main/java/com/liferay/product/navigation/personal/menu/web/internal/configuration/PersonalMenuConfigurationRegistryImpl.java:42]: cc @rafaprax

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:lima

**Comment by MrJohn1911** [comment]
ci:test:data-engine

**Comment by MrJohn1911** [comment]
ci:test:workflow

**Pull request closed without merging**